### DISCUSSION TITLE
How to download a playlist with Jellyfin metadata?

### DISCUSSION THREAD

**Discussion by FelixSchwarz** [jmbannon/ytdl-sub#990] [Q&A]
Maybe related to my question in #989 but it might be better to separate so the answers are more useful to others: I would like to download a specific Youtube playlist.

- The downloaded data should contain Jellyfin-metadata.
- I do not want to split the playlist by year, everything should be in one "season".
- Media format should be playable for common browsers/devices, mp4 would be best.
- Ideally I could organize multiple playlists in a single Jellyfin "show", each playlist being a different season.

As an example I'm looking for a config example using these two playlists:
https://www.youtube.com/playlist?list=PLfYUBJiXbdtS2UQRzyrxmyVHoGW0gmLSM
https://www.youtube.com/playlist?list=PLfYUBJiXbdtRL3FMB3GoWHRI8ieU6FhfM

I'm feeling completely lost on how to configure ytdl-sub here...

**Comment by jmbannon** [comment]
Apologies for the delayed response, what you want is https://ytdl-sub.readthedocs.io/en/latest/prebuilt_presets/tv_shows.html#tv-show-collection
